ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -145,7 +144,7 @@ const tabs = [
   { name: 'Data', link: 'data' },
   {
     name: 'Analysis',
```

Comment:
Can't use "uppercase analysisTabName" here because analysisTabName = "notebooks" when the FF is false.  We do not want to make any changes when the FF is false.